components/collapse/style/index.less

```diff
@@ -39,7 +39,7 @@
         display: inline-block;
         font-size: @font-size-sm;
         line-height: 46px;
-        transform: translateY(-50%);
+        transform: translateY(-21px);
```

Comment:
`-50%` would be -23px so by making it `-21px` we can eliminate the shift seen earlier

Reply:
Maybe adjust `line-height` instead? Use `px` instead of `%` may little hard code.